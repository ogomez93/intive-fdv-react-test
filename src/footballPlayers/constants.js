export const NAME = 'footballPlayers';
export const PLAYERS_API = 'https://football-players-b31f2.firebaseio.com/players.json?print=pretty';