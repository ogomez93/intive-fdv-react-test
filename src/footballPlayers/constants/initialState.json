{
  "fetching": true,
  "players": [],
  "nameFilter": "",
  "ageFilter": "",
  "positionFilter": ""
}
