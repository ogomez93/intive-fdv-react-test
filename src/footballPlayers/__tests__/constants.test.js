import * as constants from '../constants';

describe('Global FootballPlayers constants', () => {
  describe('NAME', () => {
    it('should match the snapshot', () =>
      expect(constants.NAME).toMatchSnapshot());
  });

  describe('PLAYERS_API', () => {
    it('should match the snapshot', () =>
      expect(constants.PLAYERS_API).toMatchSnapshot());
  });

  describe('POSITIONS', () => {
    it('should match the snapshot', () =>
      expect(constants.POSITIONS).toMatchSnapshot());
  });
});
