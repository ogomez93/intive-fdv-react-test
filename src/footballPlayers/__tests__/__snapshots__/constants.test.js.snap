// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global FootballPlayers constants NAME should match the snapshot 1`] = `"footballPlayers"`;

exports[`Global FootballPlayers constants PLAYERS_API should match the snapshot 1`] = `"https://football-players-b31f2.firebaseio.com/players.json?print=pretty"`;

exports[`Global FootballPlayers constants POSITIONS should match the snapshot 1`] = `
Array [
  "Attacking Midfield",
  "Central Midfield",
  "Centre-Back",
  "Centre-Forward",
  "Defensive Midfield",
  "Keeper",
  "Left Midfield",
  "Left Wing",
  "Left-Back",
  "Right-Back",
]
`;
